test/integration/util/Ordering.test.ts

```diff
@@ -103,7 +103,8 @@ describe('terms order', () => {
   it('dateTime type comparison', () => {
     genericOrderTestLower(dateTime('2000-01-01T00:00:00Z'), dateTime('2001-01-01T00:00:00Z'));
   });
-  it('langString type comparison', () => {
+  it.skip('langString type comparison', () => {
```

Comment:
Code relates to https://github.com/comunica/sparqlee/blob/master/lib/util/Ordering.ts#L110-L115. @rubensworks do you recall why this case exists? The spec does not say anything about this I think. Would we like to have a langstring comparison? Should we add an `Operator Extension` for it?

Reply:
Indeed, I think we added support for it according to SPARQL's operator extensibility: https://www.w3.org/TR/sparql11-query/#operatorExtensibility
We probably want to keep this in whatever form it can be implemented most easily.

Reply:
Makes sense, but if this is the case, we would need to define it in the regular function, so the inequality functions in FILTER also work accordingly. Do you agree?

Reply:
Sure, but wouldn't the ineq already work?
The table [here](https://www.w3.org/TR/sparql11-query/#OperatorMapping) seems to indicate that inequality should work for all RDF terms (so that would include langStrings).

Reply:
With inequality (ongelijkheid? Maybe I translated too literally?), I meant defining the `<` operator for it. I do not see that in the table?

Reply:
Ah, the smaller-than operator you mean, I thought you meant `!=`.
But yes, I think we want to keep it.